test/functional/android/push_file_tests.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+import desired_capabilities
+
+from appium import webdriver
+
+
+class PushFileTests(unittest.TestCase):
+    def setUp(self):
+        desired_caps = desired_capabilities.get_desired_capabilities('ApiDemos-debug.apk')
+        self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
+
+        # relax template matching
```

Comment:
it looks like the next few lines are unrelated to this test; maybe they were copy and pasted from a different example?

Reply:
Yes, we can remove below settings since they only affect image related features.

Reply:
My bad here, you're right @jlipps this is leftovers from a different example